webrtc.html

```diff
@@ -625,6 +625,313 @@
           </li>
         </ol>
 
+        <p>To <dfn id=
+        "update-ice-gathering-state">update the ice gathering state</dfn> of an
```

Comment:
You can use `data-lt` attribute for `dfn` elements and then you don't have to use `href` attributes as long as the text matches.